client/src/components/ui/OfficeSelect.svelte

```diff
@@ -1,7 +1,7 @@
 <script lang="ts">
     import type { AllOffices } from '../../../../model/src/api.ts';
     export let offices: AllOffices;
-    export let oid: number | null = null;
+    export let oid = 1;
```

Comment:
```suggestion
    export let oid: number | null = null;
```
The task of assigning an initial value to the oid falls on the parent component calling the OfficeSelect. We need to set this initially as null to force this constraint.

Reply:
resolved